addons/slack/install.js

```diff
@@ -0,0 +1,56 @@
+/*
+This is the installation logic of the Lifecycle Manager. 
+*/
+
+const Fs = require('fs');
+const Sdk = require('@fusebit/add-on-sdk');
+const Superagent = require('superagent');
+
+const getTemplateFiles = (fileNames) =>
+  fileNames.reduce((a, c) => {
+    a[c] = Fs.readFileSync(__dirname + `/template/${c}`, { encoding: 'utf8' });
+    return a;
+  }, {});
+
+module.exports = async (ctx) => {
+  // Create the Add-On Handler
+  await Sdk.createFunction(ctx, {
+    enableStorage: true,
+    configurationSerialized: `# Add-on configuration settings
+${Object.keys(ctx.body.configuration)
+  .sort()
+  .map((k) => `${k}=${ctx.body.configuration[k]}`)
+  .join('\n')}
+`,
+    nodejs: {
+      files: getTemplateFiles(['index.js', 'package.json', 'slack.js']),
+    },
+    metadata: {
+      fusebit: {
+        editor: {
+          navigationPanel: {
+            hideFiles: ['index.js'],
+          },
+        },
+      },
+      ...ctx.body.metadata,
```

Comment:
There was some discussion on Slack about injecting the template manager's URL to add-on's metadata from within install.js - what happened with it?

Reply:
Not sure, @bennbollay anything needed here? It appears to work as-is with the script